### PR TITLE
project: expand global metadata properties in rockcraft.yaml

### PR DESCRIPTION
**Pull request by cjdcordeiro**
Brings a new set of metadata properties to the rockcraft.yaml input schema.

- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?

-----

### CODE REVIEW

**Comment by cjdcordeiro** [comment]
ping @valentincanonical

File: rockcraft/project.py [hunk 1]
```diff
@@ -33,17 +33,36 @@ class ProjectValidationError(CraftError):
...
+    description: str
+    license: str
+    parts: Dict[str, Any]
+    summary: str
+    version: str
+    # optional
+    build_base: Optional[str] = pydantic.Field(alias='build-base')
     cmd: Optional[List[str]]
+    contact: Optional[List[str]]
+    documentation: Optional[str]
```
Comment (jnsgruk) [inline rockcraft/project.py:58]: Just a note that in `metadata.yaml` for charms, this field is called `docs` - may be worth sticking with it for consistency?

Comment (cjdcordeiro) [reply rockcraft/project.py:58]: sounds good. changed

**Comment by cjdcordeiro** [comment]
> issues

yep, `contact` is already aligned.

`issues` is interesting. I'm adding it to the spec for review

**Comment by cjdcordeiro** [comment]
closing for now since the attributed herein proposed are not yet in the roadmap

**Pull request closed without merging**